hw/cortexm/stm32/mcus.c

```diff
@@ -953,6 +953,19 @@ static const STM32PartInfo stm32_mcus[] = {
         .stm32 = &stm32f103x8b,
     /**/
     },
+    {
+        .name = TYPE_STM32F103C8, // STM32F103x[8B]
+        .cortexm = {
+            .flash_size_kb = 64,
+            .sram_size_kb = 20,
+
+            .svd_file_name = "STM32F103xx-qemu.json",
+            .svd_device_name = "STM32F103xx",
+        /**/
+        },
+        .stm32 = &stm32f103x8b,
```

Comment:
stm32f103x8b?

Reply:
Yes, they share the same capabilities. In fact they even share the datasheet. The difference is only flash size

Reply:
Ok